make clusters is broken

**What happened**:

I'm trying to follow the kind quick start instructions:
https://submariner.io/getting-started/quickstart/kind/

```
git clone https://github.com/submariner-io/submariner-operator
cd submariner-operator
make clusters
```

Result:

```
Downloading Makefile.dapper
Downloading Makefile.shipyard
Downloading dapper
Shipyard Dapper
Downloading Dockerfile.dapper
docker network create  -d bridge kind
Error response from daemon: network with name kind already exists
make: [Makefile.shipyard:58: clusters] Error 1 (ignored)
./.dapper  -S z -- make  clusters
ERROR: BuildKit is enabled but the buildx component is missing or broken.
       Install the buildx component to build images with BuildKit:
       https://docs.docker.com/go/buildx/
Error: No such object: submariner-operator:devel
Error: No such object: submariner-operator:devel
Error: No such object: submariner-operator:devel
Error: No such object: submariner-operator:devel
Error: No such object: submariner-operator:devel
docker: invalid reference format: repository name must be lowercase.
See 'docker run --help'.
make: *** [Makefile.shipyard:59: clusters] Error 125
```

I used kind to create some clusters manually before trying submariner.

**What you expected to happen**:

Have test clusters running.

**How to reproduce it (as minimally and precisely as possible)**:

See description

**Anything else we need to know?**:

**Environment**:
- Diagnose information (use `subctl diagnose all`):  no clusters to diagnose
- Gather information (use `subctl gather`): same
- Cloud provider or hardware configuration: same
- Install tools: not install yet
- Others: Fedora 39

Tried to delete the kind network since this seem to the the first issue:

```
$ docker network rm kind
kind

$ docker network list
NETWORK ID     NAME      DRIVER    SCOPE
c9a686bc26bb   bridge    bridge    local
b6dc82eb6574   host      host      local
83f913b2993e   none      null      local
```

Now it complains about buildx - but I did not enable BuildKit.

```
$ make clusters
docker network create  -d bridge kind
4d4cb4565cc4fcb058ec66b527ac8dec6d580856cd391bdd38d9fcd564cf9020
./.dapper  -S z -- make  clusters
ERROR: BuildKit is enabled but the buildx component is missing or broken.
       Install the buildx component to build images with BuildKit:
       https://docs.docker.com/go/buildx/
Error: No such object: submariner-operator:devel
Error: No such object: submariner-operator:devel
Error: No such object: submariner-operator:devel
Error: No such object: submariner-operator:devel
Error: No such object: submariner-operator:devel
docker: invalid reference format: repository name must be lowercase.
See 'docker run --help'.
make: *** [Makefile.shipyard:59: clusters] Error 125
```

If buildx is a requirement, it should be listed in the page.

I can reproduce this:

```
$ mv ~/.docker/cli-plugins/docker-buildx{,.disabled}
$ docker rmi submariner-operator:devel
$ make clusters
docker network create  -d bridge kind
Error response from daemon: network with name kind already exists
make: [Makefile.shipyard:58: clusters] Error 1 (ignored)
./.dapper  -S z -- make  clusters
ERROR: BuildKit is enabled but the buildx component is missing or broken.
       Install the buildx component to build images with BuildKit:
       https://docs.docker.com/go/buildx/
Error: No such object: submariner-operator:devel
Error: No such object: submariner-operator:devel
Error: No such object: submariner-operator:devel
Error: No such object: submariner-operator:devel
Error: No such object: submariner-operator:devel
docker: invalid reference format: repository name must be lowercase.
See 'docker run --help'.
make: *** [Makefile.shipyard:59: clusters] Error 125
```

As Nir suggests, the problem is that `buildx` is required, outside the Shipyard base image.

--- a/.dapper
+++ b/.dapper
@@ -75,7 +75,7 @@ buildargs=(--build-arg "ORG=${ORG}" --build-arg "PROJECT=${PROJECT}")
 gitid="$(git symbolic-ref --short HEAD 2>/dev/null | tr / _ || :)"
 gitid="${gitid:-$(git show --format=%h -s)}"
 container="$(basename "$(pwd)"):${gitid}"
-DOCKER_BUILDKIT=1 docker build -t "${container}" -f "${file}" "${buildargs[@]}" .
+docker build -t "${container}" -f "${file}" "${buildargs[@]}" .
 
 extract_var() {
     docker inspect "$1" | grep "$2" | sed -E "s/.*\"$2=(.*)\",?/\1/;q"

